.github/workflows/build-container-image.yml

```diff
@@ -0,0 +1,62 @@
+name: publish-container-image
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - created
```

Comment:
Maybe change this and restrict to these action types:
```
types: [released, prereleased]
```

So that the job does not get triggered every time some one saves a draft of a release.

Reply:
Sure, I'll update this. But in my case, I created a draft and saved it which did not trigger job.